docs/cluster-configuration.md

```diff
@@ -117,37 +117,17 @@ or new features that are not enabled by default.
 To enable a feature, add its name to the `featureGates` list and set it to `true`. Missing or `false` feature gates 
 disables the feature.
 
-### hotplugVolumes Feature Gate
-
-Set the `hotplugVolumes` feature gate in order to allow attaching a data volume to a running VMI.
-
-### withHostModelCPU Feature Gate
-
-Set the `withHostModelCPU` feature gate in order to enable support migration for VMs with host-model CPU mode
-
-Additional information: [LibvirtXMLCPUModel](https://wiki.openstack.org/wiki/LibvirtXMLCPUModel)
-
 ### withHostPassthroughCPU Feature Gate
-
 Set the `withHostPassthroughCPU` feature gate in order to allow migrating a virtual machine with CPU host-passthrough
 mode.
```

Comment:
```suggestion
mode. This can provide slightly better CPU performance, but should be enabled only when the Cluster is homogeneous from CPU HW perspective.
```

Reply:
The features gates we keep should be properly documented: each should have an adequate explanation for why an admin should use them, and when.

Reply:
This sentence is already there, a few lines below.